go/libkb/upak_loader.go

```diff
@@ -377,10 +376,15 @@ func (u *CachedUPAKLoader) loadWithInfo(arg LoadUserArg, info *CachedUserLoadInf
 		}
 	}
 
-	lock := u.locktab.AcquireOnName(ctx, g, arg.uid.String())
+	var lock *NamedLock
+	if !arg.cachedOnly {
```

Comment:
Don't need to singleflight if we're only loading cached data without calls to network.